automation/vars/bgp_dt01.yaml

```diff
@@ -17,19 +18,65 @@ vas:
             kind: Node
             resource_name: worker-3
             state: patched
+          - name: Disable rp_filters on OCP nodes
+            type: cr
+            definition:
+              spec:
+                profile:
+                  - data: |
+                      [main]
+                      summary=Optimize systems running OpenShift (provider specific parent profile)
+                      include=-provider-${f:exec:cat:/var/lib/ocp-tuned/provider},openshift
+
+                      [sysctl]
+                      net.ipv4.conf.enp7s0.rp_filter=0
+                      net.ipv4.conf.enp8s0.rp_filter=0
+                    name: openshift-no-reapply-sysctl
+                recommend:
+                  - match:
+                      # applied to all nodes except worker-3, because worker-3 has no enp8s0
+                      - label: kubernetes.io/hostname
+                        value: worker-0
+                      - label: kubernetes.io/hostname
+                        value: worker-1
+                      - label: kubernetes.io/hostname
+                        value: worker-2
+                      - label: node-role.kubernetes.io/master
+                    operand:
+                      tunedConfig:
+                        reapply_sysctl: false
+                    priority: 15
+                    profile: openshift-no-reapply-sysctl
+            api_version: tuned.openshift.io/v1
+            kind: Tuned
+            resource_name: openshift-no-reapply-sysctl
+            namespace: openshift-cluster-node-tuning-operator
+            state: present
         path: examples/dt/bgp/bgp_dt01/control-plane/nncp
         wait_conditions:
           - >-
             oc -n openstack wait nncp
             -l osp/nncm-config-type=standard
             --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
-            --timeout=60s
+            --timeout=300s
+          # wait until openstack-operator and infra-operator are ready too
+          - >-
+            oc -n openstack-operators wait pod
+            -l openstack.org/operator-name=openstack
+            --for condition=Ready
+            --timeout=300s
+          - >-
+            oc -n openstack-operators wait pod
+            -l openstack.org/operator-name=infra
+            --for condition=Ready
+            --timeout=300s
```

Comment:
This should be unneeded as of [1] and [2] merging.

[1] https://github.com/openstack-k8s-operators/openstack-operator/pull/1294
[2] https://github.com/openstack-k8s-operators/openstack-operator/pull/1292